docs/contributing-to-airbyte/README.md

```diff
@@ -72,6 +73,31 @@ The CDK currently does not support creating destinations, but it will very soon.
 
 **Please note that, at no point in time, we will ask you to maintain your connector.** The goal is that the Airbyte team and the community helps maintain the connector.
 
+### Semantic Versioning for Connectors
+
+Changes to connector behavior should always be accompanied by a version bump and a changelog entry. We use [semantic versioning](https://semver.org/) to version changes to connectors. Since connectors are a bit different from APIs, we have our own take on semantic versioning, focusing on maintaining the best user experience of using a connector.
+
+- Major: version when you make a change which requires manual intervention (update to config or configured catalog) for an existing connection to continue to succeed, or one in which data that was previously being synced will no longer be synced
+- Minor: version when you add user-facing functionality in a backwards compatible manner
+- Patch: version when you make backwards compatible bug fixes or performance improvements
+
+## Examples
```

Comment:
Any more to add? What about e.g. renaming a stream or field? Those would be breaking as they're essentially add + deletes right? 

This is an interesting format to keep track of which changes are breaking and which ones we could potentially automate so that they're no longer breaking :)

Reply:
- Adding a non-required param to spec: does that break configs?

Reply:
Maybe we should also consider users to think about backwards compatibility when e.g. renaming a parameter (still supporting the old name)

Reply:
Oh! Maybe add a note at the end like:

If you are not sure what type of change your commit is, ask us! Call this out in your PR and we are happy to help out.

Reply:
I think:

> Adding a non-required param to spec: minor?
minor!

> turning a single param into a oneOf?
I think it sort of depends on where we land [here](https://docs.google.com/document/d/1tbMeG_v29zVcza6JNhw7-oY50H7aY1AbVvZr9yzFmVw/edit#).
... but I think for now it is a breaking change.

Reply:
> Oh! Maybe add a note at the end

True, good callout! Then we can encourage them to add it to the list too :)

Reply:
Added `Trying to contribute, and don't see the change you want to make in this list? Call it out in your PR and your reviewer will help you pick the correct type of version change. Feel free to contribute the results back to this list!`

Reply:
> I think it sort of depends on where we land [here](https://docs.google.com/document/d/1tbMeG_v29zVcza6JNhw7-oY50H7aY1AbVvZr9yzFmVw/edit#).
... but I think for now it is a breaking change.

yup agreed. I'm hoping that config control messages will reduce the rows in this table that constitute major changes, but for now I think that it is indeed breaking

Reply:
@evantahler what about renaming a stream/field?

Reply:
> @evantahler what about renaming a stream/field?

I think that's breaking too.  A 'rename' is an addition and a removal until schema migrations get smarter.  I think.  CC @andyjih